internal/server/add_assurance_visit.go

```diff
@@ -2,36 +2,24 @@ package server
 
 import (
 	"fmt"
-	"net/http"
-	"strconv"
-
-	"github.com/gorilla/mux"
 	"github.com/ministryofjustice/opg-sirius-supervision-deputy-hub/internal/sirius"
+	"net/http"
 )
 
 type AddAssuranceVisit interface {
-	GetUserDetails(ctx sirius.Context) (sirius.UserDetails, error)
 	AddAssuranceVisit(ctx sirius.Context, assuranceType string, requestedDate string, userId, deputyId int) error
 }
 
 type AddAssuranceVisitVars struct {
-	Path          string
-	XSRFToken     string
-	DeputyDetails sirius.DeputyDetails
-	Error         string
-	Errors        sirius.ValidationErrors
+	AppVars
```

Comment:
the pattern in Workflow is to name the property e.g. `App AppVars` - just checking if you're embedding the struct intentionally? Is one approach better than the other?

Reply:
It depends on whether you want the properties of the struct flat or not. This is effectively replicating the existing structure, i.e. `DeputyDetails` is on the same level within the template variables as it was previously. If I had it as a named property, I would have had to update all the templates to access the property from within `App` instead.

As for whether one approach is better, your guess is as good as mine!